cypress/integration/smoke/BasicTest.ts

```diff
@@ -39,4 +39,21 @@ describe("Basic Elements", () => {
       .find("a[data-testid='tableInstances-linkInstance']")
       .should("be.visible");
   });
+
+  it("Instance header details are available", () => {
+    const instanceHeaderDetails = [
+      "Name",
+      "Description",
+      "Status",
+      "Time created",
+    ];
+    cy.get(
+      "table[data-ouia-component-id='Instances list table'] thead tr th"
+    ).should("have.length", 4);
+    cy.get(
+      "table[data-ouia-component-id='Instances list table'] thead tr th"
+    ).each((item, index) => {
```

Comment:
I appreciate this line. It is more readable and better maintaining solution than my. See https://github.com/sjamboro/sandbox-ui/blob/MGDOBR-654/cypress/integration/InstancesTest.ts#L111-L124.